tooling/Common.csproj.props

```diff
@@ -0,0 +1,39 @@
+<Project>
+  <PropertyGroup>
+    <!-- Common C# Properties -->
+    <LangVersion>9.0</LangVersion>
+    <Features>strict</Features>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+
+    <!-- Common NuGet Package Properties -->
+    <Authors>Bonsai Foundation</Authors>
+    <Copyright>Copyright © Bonsai Foundation CIC and Contributors 2011-2024</Copyright>
+    <PackageProjectUrl>https://bonsai-rx.org</PackageProjectUrl>
+    <PublishRepositoryUrl>true</PublishRepositoryUrl>
+    <PackageLicenseUrl>https://bonsai-rx.org/license</PackageLicenseUrl>
+    <PackageIconUrl>https://bonsai-rx.org/assets/images/bonsai.png</PackageIconUrl>
+
+    <!-- Always pack NuGet packages (except on CI where it'll be done separately) -->
+    <GeneratePackageOnBuild Condition="'$(ContinuousIntegrationBuild)' != 'true'">true</GeneratePackageOnBuild>
+
+    <!-- Emit modern symbol pacakges -->
```

Comment:
```suggestion
    <!-- Emit modern symbol packages -->
```

Reply:
Somehow this typo doesn't exist in Biohazrd, must've copy+pasted that one manually 😅

https://github.com/MochiLibraries/Biohazrd/blob/4dbaabfb8195853a931a21d9b6468baed9020bf0/tooling/Common.csproj.props#L22